pkg/util/policy/agent/agent.py

```diff
@@ -78,87 +70,179 @@ def route(self):
         return
 
 
+    def do_DELETE(self):
+        self.http_method = "DELETE"
+        self.decode_request()
+        self.route()
+
+
+    def do_POST(self):
+        """
+        Processes POST requests
+        extracts romana policy definition objects and passes it down for implementation
+        """
+
+        self.http_method = "POST"
+        self.decode_request()
+        self.route()
+
+        return
```

Comment:
No, it's not. It doesn't do damage either. However, there are subtle implications about this function's return values that are implied to the reader of the code. So, if you just came to the end of the function and the function never returns anything then just leave out the 'return'.

If you end a function with nothing, "return" or "return None", it's all exactly the same. But the implied meanings you can convey to the reader are a bit like this:

"return None": Indicate that the function sometimes returns useful stuff, but in this case we specifically wish to communicate 'None' as return value (to indicate a special condition, etc.)

"return": Use it like a 'break' to get out of a the function early (if the function doesn't normally return anything)

"" (nothing at the end of the function): Indicate that there is nothing that the caller of the function should evaluate, the function just ends, like a "procedure" in Pascal.

So, here there should be no 'return' at the end.